File: root/programs/VB/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Models/ViewModels/ShipperViweModel.vb
'**********************************************************************************
'* サンプル アプリ・モデル
'**********************************************************************************

' テスト用クラスなので、必要に応じて流用 or 削除して下さい。

'**********************************************************************************
'* クラス名        ：ShipperViweModel
'* クラス日本語名  ：サンプル アプリ・モデル
'*
'* 作成日時        ：2018/8/1
'* 作成者          ：棟梁 D層自動生成ツール（墨壺）, 日立 太郎
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'*
'**********************************************************************************

Namespace Models.ViewModels
    ''' <summary>
    ''' サンプル アプリ・モデル
    ''' </summary>
    <Serializable>
    Public Class ShipperViweModel

#Region "メンバ変数"

        ''' <summary>メンバ変数：ShipperID</summary>
        Private _PK_ShipperID As Nullable(Of System.Int64) ' Oracle対応 32 -> 64

        ''' <summary>プロパティ：ShipperID</summary>
        Public Property ShipperID() As Nullable(Of System.Int64) ' Oracle対応 32 -> 64
            Get
                Return Me._PK_ShipperID
            End Get
            Set
                Me._PK_ShipperID = Value
            End Set
        End Property

        ''' <summary>メンバ変数：CompanyName</summary>
        Private _CompanyName As System.String

        ''' <summary>プロパティ：CompanyName</summary>
        Public Property CompanyName() As System.String
            Get
                Return Me._CompanyName
            End Get
            Set
                Me._CompanyName = Value
            End Set
        End Property
        ''' <summary>メンバ変数：Phone</summary>
        Private _Phone As System.String

        ''' <summary>プロパティ：Phone</summary>
        Public Property Phone() As System.String
            Get
                Return Me._Phone
            End Get
            Set
                Me._Phone = Value
            End Set
        End Property

#End Region

    End Class
End Namespace

File: root/programs/CS/Samples4NetCore/Backend/MVC_Sample/MVC_Sample/Models/ViewModels/ShipperViweModel.cs
//**********************************************************************************
//* サンプル アプリ・モデル
//**********************************************************************************

// テスト用クラスなので、必要に応じて流用 or 削除して下さい。

//**********************************************************************************
//* クラス名        ：ShipperViweModel
//* クラス日本語名  ：サンプル アプリ・モデル
//*
//* 作成日時        ：2018/8/1
//* 作成者          ：棟梁 D層自動生成ツール（墨壺）, 日立 太郎
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//*
//**********************************************************************************

using System;

namespace MVC_Sample.Models.ViewModels
{
    /// <summary>自動生成Entityクラス</summary>
    [Serializable()]
    public class ShipperViweModel
    {
        #region メンバ変数

        /// <summary>メンバ変数：ShipperID</summary>
        private System.Int64? _PK_ShipperID; // Oracle対応 32 -> 64
        
        /// <summary>プロパティ：ShipperID</summary>
        public System.Int64? ShipperID  // Oracle対応 32 -> 64
        {
            get
            {
                return this._PK_ShipperID;
            }
            set
            {
                this._PK_ShipperID = value;
            }
        }

        /// <summary>メンバ変数：CompanyName</summary>
        private System.String _CompanyName;

        /// <summary>プロパティ：CompanyName</summary>
        public System.String CompanyName
        {
            get
            {
                return this._CompanyName;
            }
            set
            {
                this._CompanyName = value;
            }
        }
        /// <summary>メンバ変数：Phone</summary>
        private System.String _Phone;

        /// <summary>プロパティ：Phone</summary>
        public System.String Phone
        {
            get
            {
                return this._Phone;
            }
            set
            {
                this._Phone = value;
            }
        }

        #endregion
    }
}
